sam/variants/m2/variant.cpp

```diff
@@ -502,6 +502,11 @@ void init( void )
 
   // Initialize analogOutput module
   analogOutputInit();
+
+  analogReference(AR_DEFAULT);		// this set the default analogue reference voltage
+  analogWriteResolution(12u);		// set analogue write resolution to 12bit resolution
+  analogReadResolution(12u);		// set analogue read resolution to 12 bits resolution
```

Comment:
Where this code originated ([ArduinoCore-sam](https://github.com/arduino/ArduinoCore-sam/)) uses spaces for delimiters.  Would it be okay to stick to spaces rather than using tabs?